common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -397,3 +398,217 @@ def test_only_expected_fields_are_displayed(self):
+
+        # Delete certificate
+        certificate.delete_certificate()
+
+        self.certificates_page.visit()
+        self.assertEqual(len(self.certificates_page.certificates), 0)
+
+    def test_can_create_and_edit_signatories_of_certficate(self):
+        """
+        Scenario: Ensure that the certificates can be created with signatories and edited correctly.
+        Given I have a course without certificates
+        When I click button 'Add your first Certificate'
+        And I set new the name, description and signatory and click the button 'Create'
+        Then I see the new certificate is added and has one signatory inside it
+        When I click 'Edit' button of signatory panel
+        And I set the name and click the button 'Save' icon
+        Then I see the signatory name updated with newly set name
+        When I refresh the certificates page
+        Then I can see course has one certificate with new signatory name
+        When I click 'Edit' button of signatory panel
+        And click on 'Close' button
+        Then I can see no change in signatory detail
+        """
+        self.certificates_page.visit()
+        certificate = self.create_and_verify_certificate(
+            "New Certificate",
+            "Description of first certificate",
+            0,
+            [self.make_signatory_data('first')]
+        )
+        self.assertEqual(len(self.certificates_page.certificates), 1)
+        # Edit the signatory in certificate
+        signatory = certificate.signatories[0]
+        signatory.edit()
+
+        signatory.name = 'Updated signatory name'
+        signatory.title = 'Update signatory title'
+        signatory.organization = 'Updated signatory organization'
+        signatory.save()
+
+        self.assertEqual(len(self.certificates_page.certificates), 1)
+
+        signatory = self.certificates_page.certificates[0].signatories[0]
+        self.assertIn("Updated signatory name", signatory.name)
+        self.assertIn("Update signatory title", signatory.title)
+        self.assertIn("Updated signatory organization", signatory.organization)
+
+        signatory.edit()
+        signatory.close()
+
+        self.assertIn("Updated signatory name", signatory.name)
+
+    def test_must_supply_certificate_name(self):
+        """
+        Scenario: Ensure that validation of the certificates works correctly.
+        Given I have a course without certificates
+        And I create new certificate without specifying a name click the button 'Create'
+        Then I see error message "Certificate name is required."
+        When I set a name and click the button 'Create'
+        Then I see the certificate is saved successfully
+        """
+        self.certificates_page.visit()
+        self.certificates_page.create_first_certificate()
+        certificate = self.certificates_page.certificates[0]
+        certificate.name = ""
+        certificate.save()
+        self.assertEqual(certificate.mode, 'edit')
```

Comment:
If validation of signatory fields is not tested in a unit test, it would be good to add a bok choy test. If it is tested via unit tests (Jasmine), no need to duplicate.